lib/scenic/adapters/postgres.rb

```diff
@@ -81,6 +81,29 @@ def update_view(name, sql_definition)
         create_view(name, sql_definition)
       end
 
+      # Updates a view in the database using `CREATE OR REPLACE VIEW`.
+      #
+      # This results in a `CREATE OR REPLACE VIEW`. Most of the time the
+      # explicitness of the two step process used in {#update_view} is preferred to
+      # `CREATE OR REPLACE VIEW` because the former ensures that the view you are trying
+      # to update did, in fact, already exist. Additionally, `CREATE OR REPLACE
+      # VIEW` is allowed only to add new columns to the end of an existing
+      # view schema. Existing columns cannot be re-ordered, removed, or have
+      # their types changed. Drop and create overcomes this limitation as well.
+      #
+      # However, when there is a tangled dependency tree `CREATE OR REPLACE VIEW` can
+      # be preferable.
+      #
+      # This is typically called in a migration via {Statements#update_view_replace}.
+      #
+      # @param name The name of the view to update
+      # @param sql_definition The SQL schema for the updated view.
+      #
+      # @return [void]
+      def update_view_replace(name, sql_definition)
+        execute "CREATE OR REPLACE VIEW #{quote_table_name(name)} AS #{sql_definition};"
```

Comment:
houndci is apparently ignoring this rule in places (like the refresh materialized view esecute line above this in the file). I don't see how this is configured.

@derekprior Should this be broken into multiple lines or should we make an exception with hound or something else? If broken into multiple lines, how do you guys like to do that for your strings?